src/ngraph/builder/quantization_util.hpp

```diff
@@ -125,7 +126,37 @@ namespace ngraph
                 // s32 = f32 * std::pow(2, 31)/ max_abs32;
                 // s8 = f32 * std::pow(2, 7)/ max_abs8;
                 // s8 = s32 * std::pow(2, -24) * max_abs32 / max_abs8;
-                return make_constant(type, shape, std::pow(2, -24)) * (max_abs32 / max_abs8);
+
+                return make_constant(
+                           type, shape, std::pow(2, (output_type == element::i8) ? -24 : -23)) *
+                       (max_abs32 / max_abs8);
+            }
+
+            std::shared_ptr<Node> get_bias_scale(std::shared_ptr<Node> min_input,
```

Comment:
There are already three ways to calculate a `scale` value just for TensorFlow and just for scaled mode.  This isn't surprising to me, I guess... it's the reason why I opted for scale/bias implementation for Quantize and Dequantize.  However, is there some way to combine or group these calculations into some more common theme?  It feels very "wild wild west" right now.  How will one know which scale builders to use?  We need more documentation at a minimum.

Reply:
Still feels a little "wild west" to me, but not sure how to fix right now.  Let's look to consolidate and simplify our builders which calculate scale at some later date.